changelogs/fragments/2385-stat-filter-plugin.yml

```
@@ -0,0 +1,5 @@
+minor_changes:
+  - plugins/filter/filter_by_resource_type.py - Added new filter plugin, stat,
```

Would change this to look similar to how modules are displayed in the release notes.
```suggestion
  - filter_by_resource_type - Added new filter plugin, stat,
```